gulp/gulp.d.ts

```diff
@@ -5,9 +5,11 @@
 
 /// <reference path="../node/node.d.ts" />
 /// <reference path="../orchestrator/orchestrator.d.ts" />
+/// <reference path="../vinyl/vinyl.d.ts" />
```

Comment:
I tried that because I think you are correct, however there are other Gulp plugin definitions, such as gulp-if, that also reference Vinyl and they use the un-versioned Vinyl definition file, meaning that tests are not passing.

I'm also not sure how that will work when installing using `typings`, as it would require the end-user to also install the specific vinyl version too.